java/api/src/main/java/io/ray/api/utils/parallelactor/ParallelContext.java

```diff
@@ -0,0 +1,12 @@
+package io.ray.api.utils.parallelactor;
+
+import io.ray.api.ObjectRef;
+import io.ray.api.concurrencygroup.ConcurrencyGroup;
+import io.ray.api.function.RayFuncR;
+
+public interface ParallelContext {
+
+  <A> ParallelActor<A> createParallelActorExecutor(ParallelStrategy strategy, int parallelNum, ConcurrencyGroup[] concurrencyGroups, RayFuncR<A> ctorFunc);
+
+  <R> ObjectRef<R> submitParallelTask(ParallelActor parallelActor, int parallelIndex, RayFuncR func, Object[] args);
```

Comment:
```suggestion
  <R> ObjectRef<R> submitTask(ParallelActor parallelActor, int parallelIndex, RayFuncR func, Object[] args);
```

Because it only submits one task.